lib/util.js

```diff
@@ -161,9 +161,10 @@ function stylizeNoColor(str, styleType) {
 function arrayToHash(array) {
   var hash = Object.create(null);
 
-  array.forEach(function(val) {
+  for (var i = 0; i < array.length; i++) {
```

Comment:
v8 is smart enough these days to cache the length automatically, so this shouldn't be an issue.